kobuki_node/package.xml

```diff
@@ -13,71 +13,62 @@
   <url type="repository">https://github.com/kobuki-base/kobuki_ros</url>
   <url type="website">http://ros.org/wiki/kobuki_node</url>
```

Comment:
`<url type="website">https://index.ros.org/p/kobuki_node/github-kobuki-base-kobuki_ros</url>`

Reply:
Fixed in 9605787.